muxer/muxer.go

```diff
@@ -79,6 +77,10 @@ func checkParams(sink interface{}, sources []interface{}) error {
 		return fmt.Errorf("sink has invalid type[%s] kind[%s]", sinktype, sinktype.Kind())
 	}
 
+	if sinktype.ChanDir() == reflect.RecvDir {
+		return errors.New("sink channel is receive only, sink channels MUST be able to send")
```

Comment:
What are your thoughts about the 80 chars per line limit nowadays?

I recently discovered this is still useful in a lot of other situations, other than fit an older terminal screen. Most documentation code blocks are 80 chars wide, and also most code blocks in blogging platforms, in case you need to show a part of your code. If the browser puts a horizontal scrollbar we risk hiding part of the code.

Reply:
I tend to agree with [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments#line-length) on this one in the sense that Im not that hung up on the hard limit of 80 chars as long as it is not extremely hard to understand the code itself.

Reply:
Specifically on this case most of the chars are just space because of indentation, not sure if breaking it wont make it harder to understand or more cumbersome. Perhaps a bigger redesign could reduce the indentation level